### DISCUSSION TITLE
lsp_document_symbols to show access modifier (public/private)

### DISCUSSION THREAD

**Discussion by xixiaofinland** [ibhagwan/fzf-lua#1533] [Q&A]
I searched in Issues/Discussions on "lsp_document_symbols" but didnt' find this info, might be too easy or built-in that nobody asked?

I'd like the popup to show the tree-sitter nodes' access modifier information, i.e. pulibc/protected/private, when approporiate, such as for class, methods, properties.

It's really helpful to have this info for me on the languages I'm using.

**Comment by ibhagwan** [comment]
Unfortunately that information isn’t contained in the LSP response, that means this would require parsing of the lines/location to figure this out manually, not very effective.

**Reply by xixiaofinland** [reply]
sigh.. it's really an important info. If it's not supplied by LSP, ok..
But thanks for letting know @ibhagwan
